### PR TITLE
feat/LIVE-5914: introduce lint rule to detect untranslated strings

### PR DESCRIPTION
**Pull request by juan-cortes**
### 📝 Description

On issue #2249, it was identified that there was a requirement for detecting untranslated string literals on LLM (and possibly LLD, depending on feasibility). I took it upon myself to address this need, by reviewing the codebase and resolving all missing translations. I excluded any untranslated strings that originated from a debug menu or mock UI component, although it is worth noting that these should eventually be translated as well, albeit with a low priority.

For cases where an untranslated string must be used, as long as it conforms to the ignore pattern defined in the eslint configuration, it will be deemed acceptable. Essentially, if the string consists solely of symbols, spaces, or capital letters, it will not trigger the rule, thereby enabling the inclusion of dev buttons, placeholder text, or formatting spaces.
[rewritten by chatgpt]

### ❓ Context

- **Impacted projects**: `ledger-live-mobile` 
- **Linked resource(s)**: `https://ledgerhq.atlassian.net/browse/LIVE-5914` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo
Nothing to demo

### 🚀 Expectations to reach
App should be unchanged, it's a CI check and it currently should pass. 
Fixes https://github.com/LedgerHQ/ledger-live/issues/2249 

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/families/tron/VoteFlow/02-VoteCast.tsx [hunk 1]
```diff
@@ -221,7 +221,7 @@ export default function VoteCast({ route, navigation }: Props) {
                   }}
                 >
                   <LText semiBold style={styles.availableAmount}>
-                    text
+                    PLACEHOLDER_TEXT
```
Comment (haammar-ledger) [inline apps/ledger-live-mobile/src/families/tron/VoteFlow/02-VoteCast.tsx:224]: I guess I should know what's this coin UI is doing but... do you know what this is for, and how it is used?

**Review by haammar-ledger — APPROVED** [review]
No such issue on LLD side?

File: apps/ledger-live-mobile/src/families/tron/VoteFlow/02-VoteCast.tsx (hunk 1 above)
Comment (juan-cortes) [reply apps/ledger-live-mobile/src/families/tron/VoteFlow/02-VoteCast.tsx:224]: This is part of a Trans component and it's used for string interpolation, allowing you to use bold text in the middle of the sentence. The placeholder text is irrelevant, but the rule I introduced for untranslated strings requires them to be uppercase, that's why I replaced these.

**Comment by juan-cortes** [comment]
> No such issue on LLD side?

Probably yes, but I take my battles one at a time considering it was done out of sprint 😅

**Pull request merged**